templates/taxi/car_detail.html

```diff
@@ -6,12 +6,16 @@ <h1>
     <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
       Delete
     </a>
-
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user in car.drivers.all %}
```

Comment:
The condition `{% if user in car.drivers.all %}` might not work as expected because `car.drivers.all` is a queryset, and `user` is an object. Consider using `{% if user in car.drivers.all %}` with `car.drivers.all()` to ensure it returns a list, or use a filter method to check if the user is in the queryset.